db/migrate/20160504074011_set_report_type_from_chargeback_to_chargeback_vm_on_miq_reports.rb

```diff
@@ -0,0 +1,27 @@
+class SetReportTypeFromChargebackToChargebackVmOnMiqReports < ActiveRecord::Migration[5.0]
+  CHARGEBACK_REPORT_DB_MODEL = "Chargeback".freeze # old
+  CHARGEBACK_VM_REPORT_DB_MODEL = "ChargebackVm".freeze # new
+
+  class MiqReport < ActiveRecord::Base
+    serialize :db_options
+  end
+
+  def up
+    say_with_time('Set Chargeback to ChargebackVm on MiqReports for db column rtp_type in db_options') do
+      MiqReport.where(:db => CHARGEBACK_REPORT_DB_MODEL).all.each do |miq_report|
+        miq_report.db_options[:rpt_type] = miq_report.db = CHARGEBACK_VM_REPORT_DB_MODEL
```

Comment:
@gtanzillo  yes because there was a change where rtp_type 
https://github.com/ManageIQ/manageiq/pull/7081/files#diff-a1e3f16734c9d90e5b6e866305eab359L1171
 is determined by model from edit form which is in camelcase as ChargebackVm and ChargebackContainerProjects and before there was just "chargeback",

we are using rpt_type for determining custom method (for example)

https://github.com/ManageIQ/manageiq/pull/7081/files#diff-c9636a46ea81685e5b668e8caa43d3adR44

so maybe we should use tableize here 
https://github.com/ManageIQ/manageiq/pull/7081/files#diff-a1e3f16734c9d90e5b6e866305eab359L1171

and rename custom report chargeback methods to underscore case ? (and then I can remove downcase from this migration)

what do you think ?